CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Release History
 
+## v1.5.3
+
+> Release Date: Unreleased
+
+Enhancement:
+- Add validation for VCF 5.1.1 to documentation.
```

Comment:
I don't see this included specifically.

Reply:
the documentation states 5.1, how should i treat this that makes the most sense.  
keep as it is now, so 5.1 assumed 5.1.x are supported 
or
VMware Cloud Foundation 5.1.1
VMware Cloud Foundation 5.1
or
VMware Cloud Foundation 5.1, 5.1.1
or
VMware Cloud Foundation 5.1.x

keep in mind we will also need to account for any variation on 4.5

Reply:
I'm fine with leaving it as 5.1, 5.0, and 4.5 only.